doc/user/installation.rst

```diff
@@ -213,6 +276,12 @@ options from the list below.
    hardcoded arrays that FRR builds towards, so we need to know how big to
    make these arrays at build time.
 
+.. option:: --enable-shell-access
+
+   Turn on the ability of FRR to access some shell options( telnet/ssh/bash/etc )
```

Comment:
`shell options (telnet/ssh/bash etc.)` and period

Reply:
fixed

Reply:
nit: we still have a space after the opening parenthesis and a space before the closing parenthesis. Other than that this PR looks good to me.